openedx/adg/lms/webinars/helpers.py

```diff
@@ -277,23 +277,32 @@ def send_webinar_invitation_emails(webinar, old_webinar_data=None):
     if invite_email_recipients:
         send_webinar_emails(MandrillClient.WEBINAR_CREATED, webinar, list(invite_email_recipients))
 
-    if unregistered_members_emails:
-        send_webinar_emails(MandrillClient.WEBINAR_TEAM_INVITE, webinar, list(unregistered_members_emails))
+    member_email_recipients_with_no_platform_account = unregistered_members_emails.intersection(updated_webinar_emails)
+    if member_email_recipients_with_no_platform_account:
+        send_webinar_emails(
+            MandrillClient.WEBINAR_TEAM_INVITE, webinar, list(member_email_recipients_with_no_platform_account)
+        )
 
 
-def send_webinar_update_emails(webinar):
+def send_webinar_update_emails(webinar, old_webinar_data=None):
     """
     Send the webinar update emails to all the registered users in that webinar event whenever
     any event details are changed
 
     Args:
+        old_webinar_data (dict): Dict containing old data for webinar
         webinar (Webinar): The webinar to send update emails for
     """
+    old_webinar_emails = get_webinar_recipient_emails(old_webinar_data) if old_webinar_data else None
+
     webinar_team_emails = webinar.team_emails
     user_emails_registered_in_webinar = set(webinar.get_webinar_update_recipients_emails())
 
     emails_to_send_webinar_updates = user_emails_registered_in_webinar.union(webinar_team_emails)
```

Comment:
Similarly, let's use `|` for union.

Reply:
Same here, I think union is better for readability.